cmd/system-probe/config/config_linux.go

```diff
@@ -14,7 +14,8 @@ import (
 
 const (
 	// defaultSystemProbeAddress is the default unix socket path to be used for connecting to the system probe
-	defaultSystemProbeAddress = "/opt/datadog-agent/run/sysprobe.sock"
+	defaultSystemProbeAddress     = "/opt/datadog-agent/run/sysprobe.sock"
+	defaultGRPCSystemProbeAddress = "/opt/datadog-agent/run/grpc-sysprobe.sock"
```

Comment:
Is it possible to host the gRPC server on the same UDS? This snippet seems to indicate it is definitely possible. I ask because adding a socket file will require changes to helm charts, etc. for proper deployment.

https://github.com/mrdulin/grpc-go-cnode/blob/90c12ac0b805883667d8e545f7bcf2385813f03a/cmd/server/main.go#L124-L136